mirage/scenarios/default.ts

```diff
@@ -41,6 +42,8 @@ export default function(server: Server) {
         collectionScenario(server, currentUser);
         collectionPendingScenario(server, currentUser);
         collectionAcceptedScenario(server, currentUser);
+        collectionRejectedScenario(server, currentUser);
+        collectionRemovedScenario(server, currentUser);
```

Comment:
Let's consolidate these all into one scenario. If we want to see one of the states, I'm assuming we will want to see the other states as well.